crates/sncast/tests/utils/build_contracts.sh

```diff
@@ -15,10 +15,22 @@ if [ ! -d "${REPO_ROOT}/crates/sncast/tests/utils/devnet/bin" ]; then
   exit 1
```

Comment:
this file will be removed in https://github.com/foundry-rs/starknet-foundry/pull/2008 as it is no longer needed